jiant/models.py

```diff
@@ -736,7 +751,10 @@ def __init__(self, args, sent_encoder, vocab):
         self.vocab = vocab
         self.utilization = Average() if args.track_batch_utilization else None
         self.elmo = args.input_module == "elmo"
-        self.use_bert = bool(args.input_module.startswith("bert"))
+        self.use_pytorch_transformers = input_module_uses_pytorch_transformers(args.input_module)
+        self.project_before_pooling = not (
+            self.use_pytorch_transformers and args.transfer_paradigm == "finetune"
+        )  # Rough heuristic.
```

Comment:
nit: expand on this comment? (`# Always project before pooling, unless we're fine-tuning a transformer`)

Also seems a little weird that the transfer paradigm affects the _structure_ of the model rather than just what parts are tuned - maybe add a TODO to decouple this?